dotCMS/src/main/webapp/html/portlet/ext/useradmin/view_users_js_inc.jsp

```diff
@@ -388,12 +388,11 @@
 		var userEmail;
 		if(emailChanged){
             <%
-                 String emailRegex = Config.getStringProperty(com.dotmarketing.util.WebKeys.DOTCMS_USE_REGEX_TO_VALIDATE_EMAILS, null);
-                 if (emailRegex !=null && !emailRegex.isEmpty() && !emailRegex.equals("null")){%>
-                 var pattern=new RegExp("<%=emailRegex%>", 'gi');
-            <%}else{%>
-                 var pattern=/^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\.([a-zA-Z])+([a-zA-Z])+/;
-            <%}%>
+                 String emailRegex = Config.getStringProperty(com.dotmarketing.util.WebKeys.DOTCMS_USE_REGEX_TO_VALIDATE_EMAILS, "^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\\.([a-zA-Z])+([a-zA-Z])+");
```

Comment:
I suggest moving "^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\\.([a-zA-Z])+([a-zA-Z])+" to a more central place and make it a constant.